meson.build

```diff
@@ -0,0 +1,35 @@
+project('pantheon-camera', 'vala', 'c')
+
+app_name = 'org.pantheon.camera'
+
+i18n = import('i18n')
+
+add_global_arguments(
+    '-DGETTEXT_PACKAGE="' + 'pantheon-camera' + '"',
+    language : 'c')
+
+executable(
+    meson.project_name(),
+    'src/Application.vala',
+    'src/MainWindow.vala',
+    'src/Utils.vala',
+    'src/Backend/Settings.vala',
+    'src/Widgets/CameraView.vala',
+    'src/Widgets/HeaderBar.vala',
+    'src/Widgets/LoadingView.vala',
+    dependencies : [
+        dependency('clutter-gst-3.0'),
+        dependency('clutter-gtk-1.0'),
+        dependency('gee-0.8'),
+        dependency('gio-2.0'),
+        dependency('glib-2.0'),
+        dependency('granite'),
+        dependency('gtk+-3.0'),
+        dependency('libcanberra'),
+        meson.get_compiler('vala').find_library('posix')
+    ],
+    install : true
+)
+
```

Comment:
You should call your post install script here :)

`meson.add_install_script('meson/post_install.py')`